install/chart-checker/chart-linter.go

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Consider adding this to the `tools/` folders instead. We have similar checks there as well.